services/operatorsinfo/operatorsinfo_subgraph.go

```diff
@@ -0,0 +1,144 @@
+package operatorsinfo
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"fmt"
+
+	"github.com/Layr-Labs/eigensdk-go/crypto/bls"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/Layr-Labs/eigensdk-go/types"
+	"github.com/consensys/gnark-crypto/ecc/bn254"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/shurcooL/graphql"
+)
+
+type (
+	QueryOperatorByAddressGql struct {
+		Operator IndexedOperatorInfoGql `graphql:"operator(address: $address)"`
+	}
+	OperatorsInfoServiceSubgraph struct {
+		logger logging.Logger
+		client GraphQLQuerier
+		name   string
+	}
+	SocketUpdates struct {
+		Socket graphql.String
+	}
+	IndexedOperatorInfoGql struct {
+		Address    graphql.String
+		PubkeyG1_X graphql.String   `graphql:"pubkeyG1_X"`
+		PubkeyG1_Y graphql.String   `graphql:"pubkeyG1_Y"`
+		PubkeyG2_X []graphql.String `graphql:"pubkeyG2_X"`
+		PubkeyG2_Y []graphql.String `graphql:"pubkeyG2_Y"`
+		// Socket is the socket address of the operator, in the form "host:port"
+		SocketUpdates []SocketUpdates `graphql:"socketUpdates(first: 1, orderBy: blockNumber, orderDirection: desc)"`
+	}
+	IndexedOperatorInfo struct {
+		// PubKeyG1 and PubKeyG2 are the public keys of the operator, which are retrieved from the EigenDAPubKeyCompendium smart contract
+		PubkeyG1 *G1Point
+		PubkeyG2 *G2Point
+		// Socket is the socket address of the operator, in the form "host:port"
+		Socket string
+	}
+	G2Point struct {
+		*bn254.G2Affine
+	}
+	G1Point struct {
+		*bn254.G1Affine
+	}
+	GraphQLQuerier interface {
+		Query(ctx context.Context, q any, variables map[string]any) error
```

Comment:
are the `any`s necessary? If so, can you add comments as to how they are being used and what they are supposed to be. Hard to understand what this is used if we throwaway all type information.